crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -1278,6 +1271,9 @@ pub(crate) fn expression(expr: &Expr, checker: &Checker) {
             if checker.is_rule_enabled(Rule::Airflow3SuggestedUpdate) {
                 airflow::rules::airflow_3_0_suggested_update_expr(checker, expr);
             }
+            if checker.is_rule_enabled(Rule::Airflow3FunctionSignatureChange) {
+                airflow::rules::airflow_3_keyword_args_only_function(checker, expr);
```

Comment:
We probably can make a parent function like `airflow_3_function_signature_change_expr` and call `airflow_3_keyword_args_only_function` inside. IIRC, there're come other cases as well